### PR TITLE
Feature/54 show version via API

### PR DESCRIPTION
**Pull request by cayop**
The goal of this branch is get the version of the devicehub and the microservice tag in one unique query.

This is the query you need execute for get the info of the versions:

```
curl -H "Content-Type: application/json" -X GET http://127.0.0.1:5000/versions/
```
If you want to see the version of the tag you need start the tag service too.

for change the version you need update the ereuse_devicehub//init.py file and execute:
```
git install -e .
```

Next of restart the service you can see the new version

For the check the test you can use:
```
pytest -m "mvp" tests/test_endpoints.py
```

### CODE REVIEW

File: ereuse_devicehub/resources/versions/versions.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                 template_folder=None,
+                 url_prefix=None,
+                 subdomain=None,
+                 url_defaults=None,
+                 root_path=None,
+                 cli_commands: Iterable[Tuple[Callable, str or None]] = tuple()):
+        super().__init__(app, import_name, static_folder, static_url_path, template_folder,
+                         url_prefix, subdomain, url_defaults, root_path, cli_commands)
+
+        d = {'devicehub': '1.0a'}
```
Comment (slamora) [inline ereuse_devicehub/resources/versions/versions.py:61]: harcoded version value?

**Review by slamora — CHANGES_REQUESTED** [review]

File: ereuse_devicehub/resources/versions/versions.py (hunk 1 above)
Comment (JNadeu) [reply ereuse_devicehub/resources/versions/versions.py:61]: @cayop You could do the same as in the setup.py file to get the version directly from the init?

Comment (cayop) [reply ereuse_devicehub/resources/versions/versions.py]: The var 'd' is for put one default in the definition of the resource. But is true is better put something like this:
```
d = {'devicehub': '0.0.0'}
```
I changed now

Comment (cayop) [reply ereuse_devicehub/resources/versions/versions.py]: I change the open file for the:
```
from . import __version__
```
as suggests Santiago. I think is a good idea, more simple. I change this in dh and tag

**Review by slamora — APPROVED** [review]

**Pull request merged**